packit_service/service/events.py

```diff
@@ -514,20 +514,18 @@ def __init__(
         if build_pg:
             self.pr_id = build_pg.pr.pr_id
             self.commit_sha = build_pg.commit_sha
-            self.git_ref = (
-                self.commit_sha
-            )  # ref should be name of the branch, not a hash
             self.base_repo_name = build_pg.pr.project.repo_name
             self.base_repo_namespace = build_pg.pr.project.namespace
             # FIXME: hardcoded, move this to PG
             https_url = f"https://github.com/{self.base_repo_namespace}/{self.base_repo_name}.git"
+            git_ref = self.commit_sha  # ref should be name of the branch, not a hash
```

Comment:
Moving down does not help, but the rewriting will help definitely.

An alternative solution can be to move the `super` call up and safely write these values after that. What do you think?

Reply:
yes, that was my first idea, but we need to get `https_url` from the db info as it is passed to the `super` constructor

Reply:
Good point. Then maybe only the `https_url` before the `super` calls.

(Feel free to merge it as is. I just don't like a lot of code before the super call. It usually leads to this kind of weird errors...)

Reply:
I will leave it as it is for now, as here `https_url = f"https://github.com/{self.base_repo_namespace}/{self.base_repo_name}.git"` we need other attributes anyway.